packages/web-components/src/components/rux-pop-up/rux-pop-up.tsx

```diff
@@ -141,7 +152,7 @@ export class RuxPopUp {
 
     componentDidRender() {
         if (this.open) {
-            this.handleOpen()
+            this._startPositioner()
```

Comment:
This is a revert of the revert - handleOpen handles the emission of `ruxpopupopened`, so calling it here caused the extra events to be emitted. 
The reason this was changed to be handleOpen in a prev PR was so that the pop-up would show up in the correct spot and retain the ability to close on an off click if the pop-up was open by default. @FMorrison87 lemme know if that's incorrect

Reply:
That's correct yeah.